abs.yaml

```diff
@@ -0,0 +1,4 @@
+upload_channels:
+  - sfe1ed40
+channels:
+  - sfe1ed40
```

Comment:
```suggestion
```

you don't need this https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/01_How_tos/Building_Snowflake_packages.md#pull-request

Reply:
if `for_sf = True` in the `meta.yaml` this file should be removed after the merge.

Reply:
Will reply in Slack